doltpy/cli/tests/fixtures.py

```diff
@@ -0,0 +1,26 @@
+import os
+from doltpy.cli.dolt import Dolt
+import pytest
+import shutil
+from doltpy.cli.tests.helpers import get_repo_path_tmp_path
+
+
+@pytest.fixture
+def init_empty_test_repo(tmp_path) -> Dolt:
+    repo_path, repo_data_dir = get_repo_path_tmp_path(tmp_path)
+    assert not os.path.exists(repo_data_dir)
+    repo = Dolt.init(repo_path)
+    yield repo
+    if os.path.exists(repo_data_dir):
+        shutil.rmtree(repo_data_dir)
```

Comment:
maybe add a fail-safe?
try: -> make dolt
finally: -> remove repo

Reply:
The solution here in fact is to use the fact `pytest` will wipe out `tmp_path` which in turn means no `yield` syntax is needed, which in turn means these two functions can be nicely factored down.